### PR TITLE
Update tidb-lightning-distributed-import.md

### PR DESCRIPTION
**Pull request by lance6716**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [x] v6.4 (TiDB 6.4 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/issues/40818

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 1]
```diff
@@ -184,4 +184,4 @@ type = "sql"
 
 ### 导入过程中报错 "Target table is calculating checksum. Please wait until the checksum is finished and try again"
 
-在部分并行导入场景，如果表比较多或者一些表的数据量比较小，可能会出现当一个或多个任务开始处理某个表的时候，此表对应的其他任务已经完成，并开始数据一致性校验。此时，由于数据一致性校验不支持写入其他数据，对应的任务会返回 "Target table is calculating checksum. Please wait until the checksum is finished and try again" 错误。等校验任务完成，重启这些失败的任务，报错会消失，数据的正确性也不会受影响。
\ No newline at end of file
+在部分并行导入场景，如果表比较多或者一些表的数据量比较小，可能会出现当一个或多个任务开始处理某个表的时候，此表对应的其他任务已经完成，并开始数据一致性校验。此时，由于数据一致性校验不支持写入其他数据，对应的任务会返回 "Target table is calculating checksum. Please wait until the checksum is finished and try again" 错误。等校验任务完成，重启这些失败的任务，报错会消失，数据的正确性也不会受影响。
```
Comment (gozssky) [inline tidb-lightning/tidb-lightning-distributed-import.md:187]: What is changed on this line?

**Review by gozssky — APPROVED** [review]

File: tidb-lightning/tidb-lightning-distributed-import.md (hunk 1 above)
Comment (lance6716) [reply tidb-lightning/tidb-lightning-distributed-import.md:187]: maybe the file ends with an empty line. GitHub web editor did it.

File: tidb-lightning/tidb-lightning-distributed-import.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ TiDB Lightning 并行导入可以用于以下场景：
 
 ## 使用说明
 
-使用 TiDB Lightning 并行导入无须额外配置。TiDB Lightning 在启动时，会在下游 TiDB 中注册元信息，并自动检测是否有其他实例向目标集群导入数据。如果有，则自动进入并行导入模式。
+使用 TiDB Lightning 并行导入需要设置 `incremental-import = true`。TiDB Lightning 在启动时，会在下游 TiDB 中注册元信息，并自动检测是否有其他实例向目标集群导入数据。如果有，则自动进入并行导入模式。
```
Comment (JaySon-Huang) [inline tidb-lightning/tidb-lightning-distributed-import.md:27]: "并行导入需要设置 `incremental-import = true`" 添加到上文的“注意”里面？

Comment (JaySon-Huang) [reply tidb-lightning/tidb-lightning-distributed-import.md:27]: “示例 2：使用 TiDB Lightning 并行导入单表数据” 里面的 toml 文件也加上`incremental-import = true` 这个配置项会好一些

Comment (lance6716) [reply tidb-lightning/tidb-lightning-distributed-import.md:27]: https://github.com/pingcap/docs-cn/pull/12828/commits/e0da389d59afcd553aedcda077962ca37d3f3619

**Review by ran-huang — APPROVED** [review]

**Review by JaySon-Huang — APPROVED** [review]

**Pull request merged**